src/preset-themes/001.json

```diff
@@ -2,44 +2,46 @@
   "title": "001",
   "colors": {
     "toolbar": {
-      "h": 187,
-      "s": 77,
-      "l": 53,
-      "a": 100
+      "r": 43,
+      "g": 206,
+      "b": 227,
+      "a": 1
     },
     "toolbar_text": {
-      "h": 213,
-      "s": 43,
-      "l": 89
+      "r": 215,
+      "g": 226,
+      "b": 239
     },
     "accentcolor": {
-      "h": 187,
-      "s": 77,
-      "l": 53
+      "r": 43,
+      "g": 206,
+      "b": 227
     },
     "textcolor": {
-      "h": 187,
-      "s": 0,
-      "l": 33
+      "r": 84,
+      "g": 84,
+      "b": 84
     },
     "toolbar_field": {
-      "h": 211,
-      "s": 0,
-      "l": 100,
-      "a": 53.33333206176758
+      "r": 255,
+      "g": 255,
+      "b": 255,
+      "a": 1
```

Comment:
I feel like this is weird and wrong. Shouldn't the "a" value here be something like "0.533"?